src/_cost-storage.ts

```
@@ -0,0 +1,204 @@
+			catch: error => error as Error,
+		})),
+	);
+}
+
+/**
+ * Load cost data for a session
+ * @param sessionId - Session ID
+ * @returns Result with cost data or error if not found/invalid
+ */
+export async function loadCostData(sessionId: string): Promise<Result<CostData, Error>> {
+	return Result.try({
```

ESLint

[error] Unsafe call of a(n) `error` type typed value.

(ts/no-unsafe-call)